mealpy.py

```diff
@@ -31,10 +33,16 @@ def __init__(self):
         self.cities = None
         self.schedules = None
 
-    def login(self, username, password):
-        data = {'username': username, 'password': password}
-        r = requests.post(
-            LOGIN_URL, data=json.dumps(data), headers=self.headers)
+    def login(self):
+        email = input('Enter email: ')
+        password = getpass.getpass('Enter password: ')
```

Comment:
Just read this issue https://github.com/edmundmok/mealpy/issues/10. 
I think this can be solved by separating the script/scheduler part from the `MealPal` class part, instead of adding the input within this `login` method, since this restricts the login mechanism to be through command line input. Also relevant to your update on https://github.com/edmundmok/mealpy/issues/6.

Reply:
Can you elaborate, do you mean passing the creds through the constructor of `Mealpal`? Then it can be loaded via cli prompt or read env var?

Btw, I'm not well-versed in security but my thinking was to release the creds as soon as we get can get a cookie.  For scheduled runs, we could keep the session alive by periodically prodding the site, extending the TTL (I think, I don't know how session cookies ttl is supposed to work).

Reply:
> Can you elaborate, do you mean passing the creds through the constructor of Mealpal? Then it can be loaded via cli prompt or read env var?

Just from a software engineering POV, there will be a dependency on how the input is being read, as opposed to the current method where the credentials can be passed in through the parameters. I was also wondering if the credentials can be stored somewhere to be read from, but that sounds really bad in terms of security.

> For scheduled runs, we could keep the session alive by periodically prodding the site, extending the TTL (I think, I don't know how session cookies ttl is supposed to work).

That sounds possible, depending on whether the server returns an updated cookie, I think I've used some APIs that do that.